### ISSUE TITLE
build openface fail

### ISSUE THREAD

**Issue by Vincechen611** [cmusatyalab/openface#71]
I try to build openface on docker but fail.
The resource is removed from http://archive.ubuntu.com/ubuntu/dists/trusty-updates/main/binary-amd64/Packages

Here is the  exception message...

Get:15 http://archive.ubuntu.com trusty-security/universe amd64 Packages [158 kB]
Get:16 http://archive.ubuntu.com trusty Release [58.5 kB]
Get:17 http://archive.ubuntu.com trusty/main Sources [1335 kB]
Get:18 http://archive.ubuntu.com trusty/restricted Sources [5335 B]
Get:19 http://archive.ubuntu.com trusty/universe Sources [7926 kB]
Get:20 http://archive.ubuntu.com trusty/main amd64 Packages [1743 kB]
Get:21 http://archive.ubuntu.com trusty/restricted amd64 Packages [16.0 kB]
Get:22 http://archive.ubuntu.com trusty/universe amd64 Packages [7589 kB]
Fetched 21.4 MB in 1min 42s (210 kB/s)
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/trusty-updates/main/binary-amd64/Packages  Hash Sum mismatch

E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c curl -s https://raw.githubusercontent.com/torch/ezinstall/master/install-deps | bash -e' returned a non-zero code: 100

**Comment by bamos** [comment]
Hi @Vincechen611 - I just successfully built a container from scratch, which makes it improbable there was a backward-incompatible change to Torch or one of it's dependencies. There's a small chance this was a transient issue and has been resolved. Otherwise, I'm not sure how to help you further debug this. We might have to get the Torch community to help with this.

What system are you building this in?

I'm hoping to get a Docker automated build out over the next few weeks so users won't have to build a container from scratch. I'm tracking the progress in #35 and will comment here when it's uploaded if you want to try it.

-Brandon.
